lib/include/hse_ikvdb/csched.h

```diff
@@ -22,6 +21,40 @@ struct mpool;
 struct hse_kvdb_compact_status;
 struct kvdb_health;
 
+/* clang-format off */
+
+/* work queues */
+#define SP3_QNUM_ROOT           (0)
```

Comment:
Should this maybe be an enum instead?

Reply:
Probably, but I'll leave it be for this patch, too much churn...